docs/sources/enterprise/saml/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'SAML authentication'
+description: 'Grafana SAML authentication'
+keywords: ['grafana', 'saml', 'documentation', 'saml-auth']
```

Comment:
Do we need to add "enterprise" keyword also?

Reply:
hmm, let me see if we have that

Reply:
your right, it looks like most if not all enterprise docs have the enterprise keyword.

I'l add it